src/components/become-tutor-steps/general-info/GeneralInfo.js

```diff
@@ -0,0 +1,78 @@
+import { useTranslation } from 'react-i18next'
+import { Box, Checkbox, FormControlLabel, Typography } from '@mui/material'
+
+import AppTextField from '~/components/app-text-field/AppTextField'
+
+import img from '~/assets/img/mentor-home-page/become-tutor/general-info.png'
+import { styles } from './general-info.styles'
+
+const GeneralInfo = ({ data, handleChange, handleBlur, errors, btnsBox }) => {
```

Comment:
It is not a component. It is the container. Move it to the appropriate folder.

Reply:
fixed